### PR TITLE
Edge cie 5018 remote software management

### PR DESCRIPTION
**Pull request by TzviLipschitz**
Duplicate PR should be created later to merge into develop/r2024

### CODE REVIEW

File: content/edge-k8s/k8-edge-connecting-edge-to-cloud-bundle/software-management-docs.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+weight: 20
+title: Managing Software Remotely
```
Comment (BeateRixen) [inline content/edge-k8s/k8-edge-connecting-edge-to-cloud-bundle/software-management-docs.md:3]:
```suggestion
title: Managing software remotely
```

File: content/edge-k8s/k8-edge-connecting-edge-to-cloud-bundle/software-management-docs.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+When using the remote software management feature, the manifest for applications, microservices and extensions must contain a _name_, _key_, _version_ and _contextPath_. These values are used when installing the software rather than using the name and version from the software repository. The name and version from the manifest are also listed on the devices **Software** tab when installed.
+{{< /c8y-admon-info >}}
+
+### Installing software
+1. Navigate to your Edge device's **Software** tab. The **Software** tab lists your installed applications, microservices and extensions. Click **Install software**. The **Install software** dialog lists the software from the software repository which matches Edge's supported types of _c8yedge_application_, _c8yedge_microservice_ and _c8yedge_extension_.
+2. Select one or multiple software items.
+3. Click **Install**.
+4. Under **Software changes** on the right, review your planned changes and confirm the software update operation by clicking **Apply changes**.
+
+The install operation will be created and executed on the device. The software installation is completed as soon as the device has installing the software.
```
Comment (jmshark) [inline content/edge-k8s/k8-edge-connecting-edge-to-cloud-bundle/software-management-docs.md:27]:
```suggestion
The install operation will be created and executed on the device. The software installation is completed when the device has installed the software.
```

File: content/edge-k8s/k8-edge-connecting-edge-to-cloud-bundle/software-management-docs.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+{{< c8y-admon-info >}}
+Applications and microservices will be installed and subscribed to on your 'edge' tenant but extensions will only be installed.
+{{< /c8y-admon-info >}}
+
+### Updating software
+To update software on your Edge device, hover over the software item which you want to update and click **Update**. Select a version from the list and click **Update** again. The software will be updated with the selected version.
+
+### Removing software
+To remove software on your Edge device, hover over the software item which you want to delete and click the delete icon.
```
Comment (jmshark) [inline content/edge-k8s/k8-edge-connecting-edge-to-cloud-bundle/software-management-docs.md:37]:
```suggestion
To remove software on your Edge device, hover over the software item which you want to remove and click the remove icon.
```
The terminology here and the button/icon hover text says remove. Although frustratingly the smartrest is a delete :(

image

File: content/edge-k8s/k8-edge-connecting-edge-to-cloud-bundle/software-management-docs.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+4. Under **Software changes** on the right, review your planned changes and confirm the software update operation by clicking **Apply changes**.
+
+The install operation will be created and executed on the device. The software installation is completed as soon as the device has installing the software.
+
+{{< c8y-admon-info >}}
+Applications and microservices will be installed and subscribed to on your 'edge' tenant but extensions will only be installed.
+{{< /c8y-admon-info >}}
+
+### Updating software
+To update software on your Edge device, hover over the software item which you want to update and click **Update**. Select a version from the list and click **Update** again. The software will be updated with the selected version.
```
Comment (jmshark) [inline content/edge-k8s/k8-edge-connecting-edge-to-cloud-bundle/software-management-docs.md:34]: In the install we talk about the operation being created and executing to completion but we don't do that for that update or remove, we should be consistent.

**Review by MWindel — APPROVED** [review]

**Pull request merged**